File: python/tools/latn_iast_transliteration_verification/deva_to_latn_iast_test.py
from __future__ import annotations

from indic_script_converter.brahmic_to_latn_iast import to_canonical_iast_from_devanagari
from tools.fixtures.example_deva import devanagari_smoke_samples


def main() -> None:
    print('-' * 64)
    print(' DEVANAGARI TO LATN IAST TRANSLITERATION')
    print('-' * 64)
    for source in devanagari_smoke_samples:
        result = to_canonical_iast_from_devanagari(source)
        print(f'"{source}" -> "{result}"')


if __name__ == '__main__':
    main()

File: javascript/tools/latn_iast_transliteration_verification/deva-to-latn-iast-test.js
import { devanagariSmokeSamples, toCanonicalIastFromDevanagari } from '../../src/index.js';

export function main() {
    console.log('----------------------------------------------------------------');
    console.log(' DEVANAGARI TO LATN IAST TRANSLITERATION');
    console.log('----------------------------------------------------------------');
    for (const source of devanagariSmokeSamples) {
        console.log(`"${source}" -> "${toCanonicalIastFromDevanagari(source)}"`);
    }
}

if (import.meta.url === `file://${process.argv[1]}`) main();
